build.gradle

```diff
@@ -75,6 +75,8 @@ subprojects { project ->
                 check("MissingBraces", CheckSeverity.ERROR)
                 check("TypeToString", CheckSeverity.ERROR)
                 check("SymbolToString", CheckSeverity.ERROR)
+                // for doing auto-patching
+//                errorproneArgs.addAll("-XepPatchChecks:MissingSummary", "-XepPatchLocation:IN_PLACE")
```

Comment:
Do we want to keep this around? I would think this is a one time config...

Reply:
I thought it might be handy as an example in case we want to auto-patch in the future.  I can extend the comment to clarify its purpose and give full instructions (`-Werror` also needs to be removed).  Alternately I can just delete; WDYT?

Reply:
Either works for me, but if we are leaving it as an example, then I'd indeed add a slightly longer comment/instructions and maybe just replace `MissingSummary` for `[CheckerName]`?

Reply:
Fixed in [1ff9f9f](https://github.com/uber/NullAway/pull/707/commits/1ff9f9f0cef0ac861afe23fb1e17f84c2bb8bd35)